pkg/ifuzz/ifuzz_test.go

```diff
@@ -116,6 +116,47 @@ func testDecode(t *testing.T, arch string) {
 	}
 }
 
+func TestGenerate(t *testing.T) {
+	for _, arch := range allArches {
+		t.Run(arch, func(t *testing.T) {
+			testGenerate(t, arch)
+		})
+	}
+}
+
+func testGenerate(t *testing.T, arch string) {
+	insnset := iset.Arches[arch]
+	r := rand.New(testutil.RandSource(t))
+	for mode := iset.Mode(0); mode < iset.ModeLast; mode++ {
+		for repeat := 1; repeat < 10; repeat++ {
+			cfg := &iset.Config{
+				Arch: arch,
+				Mode: mode,
+				Priv: true,
+				Exec: true,
+				Len: repeat,
+			}
+			if len(insnset.GetInsns(mode, iset.TypeUser)) == 0 {
```

Comment:
Can this be done right after the following line?
```
for mode := iset.Mode(0); mode < iset.ModeLast; mode++ {
```

Reply:
Sure. By the way, do we want to hide this behind something like `insnset.HasInsns(mode)`?

Reply:
If it's used only in this test, I don't have strong preference.